core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -162,7 +167,10 @@ def __init__(self, plugindef):
         self.start_fast = False
         self.stop_wait_wal_archive = True
         self.switch_wal = True
+        # keep the timeout in sync with your checkpoint timeout
```

Comment:
I agree This comment needs to be improved.

It doesn't make sense to have this parameter for example lower than either db checkpoint_timeout and/or archive_timeout, because they defined how much time archiving might occur.

[archive_timeout](https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-ARCHIVE-TIMEOUT)

Reply:
Done by adding 
```
# keep the timeout in sync with your PG checkpoint/archive timeout parameters
```